examples/webpack-demo-vanilla-bundle/src/examples/example07.html

```diff
@@ -54,6 +54,10 @@ <h3 class="title is-3">
     <span class="icon mdi mdi-close"></span>
     <span>Clear Filters</span>
   </button>
+  <button class="button is-small" data-test="clear-filters-btn" onclick.delegate="allFilters()">
```

Comment:
you need to change the `data-test="clear-filters-btn"` to something else, this is used in the Cypress E2E tests and this is what makes Cypress fail now. I guess `data-test="modal-filters-btn"` would be ok